integration/simulation/network/geth_network.go

```diff
@@ -115,11 +115,15 @@ func (n *networkInMemGeth) Create(params *params.SimParams, stats *stats.Stats)
 func (n *networkInMemGeth) TearDown() {
 	for _, client := range n.obscuroClients {
 		temp := client
-		go (*temp).Call(nil, obscuroclient.RPCStopHost) //nolint:errcheck
-		go (*temp).Stop()
+		go func() {
+			_ = (*temp).Call(nil, obscuroclient.RPCStopHost)
+			(*temp).Stop()
```

Comment:
@BedrockSquirrel Would it be better to do:

```
defer (*temp).Stop()
_ = (*temp).Call(nil, obscuroclient.RPCStopHost)
```

That way, if RPCStopHost blows up, we'll still stop the client.